dockerfile/anaconda-rpm/Dockerfile

```diff
@@ -52,6 +51,7 @@ RUN set -ex; \
     dnf copr enable -y ${copr_repo} ${CI_TAG}-x86_64; \
     dnf copr enable -y @storage/blivet-daily ${CI_TAG}-x86_64; \
     dnf copr enable -y @storage/udisks-daily ${CI_TAG}-x86_64; \
+    dnf install -y python3-polib; \
```

Comment:
The commit message is not alligning with what I read in the commit. You move the python3-polib to not be installed in ELN at all, not be installed from pip.